TLS_ADHERENCE_TEST_PLAN.md

```
@@ -0,0 +1,211 @@
+oc delete pod -n openshift-operators -l app.kubernetes.io/name=devworkspace-controller
+
+# Wait for new pod
+sleep 10
+
+# Check logs
+CONTROLLER_POD=$(oc get pods -n openshift-operators -l app.kubernetes.io/name=devworkspace-controller -o jsonpath='{.items[0].metadata.name}')
+oc logs -n openshift-operators $CONTROLLER_POD -c devworkspace-controller | grep -A3 "Applying cluster TLS profile"
+```
+
+**Expected**: Log shows:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)